### ISSUE TITLE
Execute sync and async scripts

### ISSUE THREAD

**Issue by voxsim** [detro/ghostdriver#526]
I open this pull request because ghost driver does not fully support  this spec: https://w3c.github.io/webdriver/webdriver-spec.html#dependencies.
For what I understand to execute a sync script we need to do a post to /execute/sync instead of /execute and to execute an async script we need to do a post to /execute/async instead of /execute_async.

In detail:
https://w3c.github.io/webdriver/webdriver-spec.html#dfn-execute-script
https://w3c.github.io/webdriver/webdriver-spec.html#dfn-execute-async-script

Do you think it will be useful a pull request about this? Do you think to support this spec?

thanks in advance,
cheers,
Simon

**Comment by jesg** [comment]
> Do you think it will be useful a pull request about this?

yes.  make sure you don't remove the old [api](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol) since it is what most of the client drivers use.  i need to run the full regression suite to approve this kind of change so it will take some time for me to merge it in.

> Do you think to support this spec?

yes we should support everything we can in the w3c spec.

**Comment by voxsim** [comment]
ok I will work on this ;)
